python/llm/example/GPU/HuggingFace/Multimodal/qwen2-audio/generate.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+from io import BytesIO
+from urllib.request import urlopen
+import librosa
+import torch
+from transformers import Qwen2AudioForConditionalGeneration, AutoProcessor
+from ipex_llm import optimize_model
+
+def main(args):
+    model_path = args.repo_id_or_model_path
+
+    processor = AutoProcessor.from_pretrained(model_path)
+    model = Qwen2AudioForConditionalGeneration.from_pretrained(model_path)
+    model = optimize_model(model, low_bit='sym_int4', optimize_llm=True)
+    model = model.half()
+    model = model.to('xpu')
+
+    conversation = [
+        {"role": "user", "content": [
+            {"type": "audio", "audio_url": "https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen2-Audio/audio/guess_age_gender.wav"},
+        ]},
+        {"role": "assistant", "content": "Yes, the speaker is female and in her twenties."},
+        {"role": "user", "content": [
+            {"type": "audio", "audio_url": "https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen2-Audio/audio/translate_to_chinese.wav"},
+        ]},
```

Comment:
can we just keep the last user content as conversation? and make "https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen2-Audio/audio/translate_to_chinese.wav" as a parameter ?